minifi-assembly/NOTICE

```diff
@@ -681,6 +681,14 @@ The following binary components are provided under the Common Development and Di
     (CDDL 1.0) (GPL3) Streaming API For XML (javax.xml.stream:stax-api:jar:1.0-2 - no url provided)
 
 ************************
+Common Public License 1.0
+************************
+
+The following binary components are provided under the Common Public License Version 1.0.  See project link for details.
```

Comment:
I believe this is unneeded as it is just a test dependency and neither the source of this project nor its binary representation make its way into any of the generated distributions (source or convenience binaries).